test/e2e/framework.go

```diff
@@ -1142,6 +1148,41 @@ func (data *TestData) GetEncapMode() (config.TrafficEncapModeType, error) {
 	return config.TrafficEncapModeInvalid, fmt.Errorf("antrea-conf config map is not found")
 }
 
+func (data *TestData) getFeatures(confName string, antreaNamespace string) (featuregate.FeatureGate, error) {
+	featureGate := featuregate.NewFeatureGate()
+	if err := featureGate.Add(features.DefaultAntreaFeatureGates); err != nil {
```

Comment:
Why not using `DefaultMutableFeatureGate` which is for mutating FeatureGate? I think `DefaultAntreaFeatureGates` doesn't need to be public in that way.

Reply:
I thought it may be not good to reuse a global variable. I also have an idea that uses a function to copy the `defaultAntreaFeatureGates` without returning it directly to keep it read-only.

Reply:
Thanks for the explanation. Then perhaps you could just use `DefaultMutableFeatureGate.DeepCopy()` to avoid constructing the featuregate from scratch again.

Reply:
Thanks, updated.